client/js/constants/genus_types.js

```diff
@@ -24,14 +24,7 @@ export const types = {
 };
 
 export function getQbankType(type) {
-  switch (type) {
-    case types.item.multipleChoice:
-      return 'multipleChoice';
-
-    case types.default:
-    default:
-      return null;
-  }
+  return _.findKey(types.item, (genusType) => type === genusType) || null;
```

Comment:
which may be ok, I just figured I'd leave it returning what you had the switch returning